### PR TITLE
26 awaiting reveiw + article page

### PR DESCRIPTION
**Pull request by fadeomar**
#### Issues this PR relates to
#26 
=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]
- [x] `Continue reading...` font weight should be 700
- [ ] spacing between each articles should be 80px
- [x] in article page, date font size doesn't match figma
image
- [ ] date component should has higher z index 
image
 - [ ] return home button should take me to awaiting review list again

File: client/src/pages/ArticlePage/index.js [hunk 1]
```diff
@@ -10,25 +10,73 @@ import {
...
+
+  const onReject = () => {
+    setLoading(true);
+    try {
+      const { data, error } = rejectArticle({ id });
+      if (error) {
+        setPageError(error.message);
+      } else {
+        // eslint-disable-next-line no-console
+        console.log({ data });
```
Comment (RamyAlshurafa) [inline client/src/pages/ArticlePage/index.js:50]:
```suggestion
```

File: client/src/pages/ArticlePage/index.js [hunk 2]
```diff
@@ -10,25 +10,73 @@ import {
...
+
+  const onApprove = () => {
+    setLoading(true);
+    try {
+      const { data, error } = approveArticle({ id });
+      if (error) {
+        setPageError(error.message);
+      } else {
+        // eslint-disable-next-line no-console
+        console.log({ data });
```
Comment (RamyAlshurafa) [inline client/src/pages/ArticlePage/index.js:68]:
```suggestion
```

File: client/src/pages/EditEventPage/style.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import styled from '@emotion/styled';
+
+export const Form = styled.form``;
```
Comment (RamyAlshurafa) [inline client/src/pages/EditEventPage/style.js:3]: i cant see where this been used

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]
- [ ] buttons need right padding on smaller screens
image

File: client/src/pages/EditEventPage/style.js (hunk 3 above)
Comment (fadeomar) [reply client/src/pages/EditEventPage/style.js:3]: C:\Users\Lenovo\Desktop\code\working-class-history\client\src\pages\EditEventPage\ContributeForm.js

File: client/src/pages/ArticlePage/index.js (hunk 2 above)
Comment (fadeomar) [reply client/src/pages/ArticlePage/index.js:68]: done

File: client/src/pages/ArticlePage/index.js (hunk 1 above)
Comment (fadeomar) [reply client/src/pages/ArticlePage/index.js:50]: done

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]
image

**Review by RamyAlshurafa — APPROVED** [review]

**Pull request merged**